spec/features/nominate_practices/nominate_a_practice_page_spec.rb

```diff
@@ -30,16 +30,24 @@
       expect(page).to have_content('Message sent. The Diffusion Marketplace team will review your nomination.')
     end
 
-    it 'should redirect the user to /nominate-an-innovation if they try to visit the old /nominate-a-practice URL' do
-      visit '/nominate-a-practice'
-      expect(page).to have_current_path(nominate_an_innovation_path)
+    it 'should log and redirect user to homepage if phone field is populated' do
+      fill_in('Your email address', with: 'test@test.com')
+      fill_in('Subject line', with: 'Test subject')
+      # all fields should be required
+      click_button('Send message')
+      message = find('#message').native.attribute('validationMessage')
+      expect(message).to eq('Please fill out this field.')
+      fill_in('Your message Please include information about where your innovation is being implemented.', with: 'This is a test message')
+      fill_in('phone', with: 'this is spam')
+      # make sure the mailer count does not increase
+      expect { click_button('Send message') }.to change { ActionMailer::Base.deliveries.count }.by(0)
+      # make sure user is redirected to home page.
+      expect(page).to have_current_path(root_path)
```

Comment:
Can we add an expectation here to ensure a `FormSpam` record is being created?

Reply:
Added expect: expect { click_button('Send message') }.to change(FormSpam, :count).by(1)